config/200-config.yaml

```
@@ -80,3 +80,7 @@ data:
     # NOTE THAT THIS IS AN EXPERIMENTAL / ALPHA FEATURE.
     enable-cryptomb: "false"
 
+
+    # trusted-hops-count Configure the number of additional ingress proxy hops from the
```

```suggestion
    # Configures the number of additional ingress proxy hops from the
```

We should align the format with the other values.